website/docs/install/install-with-script.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Quickstart
 
 <table>
     <tr>
-    <th style={{minWidth: 180 + 'px'}}>Goal</th> 
+        <th style={{minWidth: 180 + 'px'}}>Node type</th> 
+        <th>Description</th>
+    </tr>
+    <tr>
+      <td><strong>Ethereum node</strong><br />"Node"</td>
+      <td>An Ethereum node is an <strong>execution node</strong> and <strong>beacon node</strong> working together. Ethereum nodes communicate peer-to-peer to secure the Ethereum network, and require both <strong>execution-layer client software</strong> and <strong>consensus-layer client software</strong>.</td>
+    </tr> 
+    <tr>
+      <td><strong>Execution node</strong></td>
+      <td>Execution nodes use execution client software to process transactions and smart contracts in Ethereum's execution layer. Examples of execution client software include Nethermind, Besu, and Geth. Execution nodes connect to each other, and to beacon nodes.</td>
+    </tr>
+    <tr>
+      <td><strong>Beacon node</strong></td>
+      <td>Beacon nodes use beacon node client software to coordinate Ethereum's proof-of-stake consensus mechanism in Ethereum's consensus layer. Prysm, Teku, Lighthouse, and Nimbus are examples of consensus-layer clients that contain both beacon node and validator node client software. Beacon nodes connect to each other, to execution nodes, and to validator nodes.</td>
+    </tr>
+    <tr>
+      <td><strong>Validator node</strong><br/> "Validator"</td>
+      <td>Validator nodes let people stake 32 ETH as collateral within Ethereum's consensus layer. Validator nodes are responsible for proposing blocks within Ethereum's proof-of-stake consensus mechanism, and will fully replace proof-of-stake miners after The Merge. Validator nodes connect to each other, and to beacon nodes.</td>
+    </tr>
+</table>
+
+
+<br />
+
+
+<img style={{width: 100 + '%', margin: 'auto', display: 'block', maxWidth: 651 + 'px'}} src={ClientStackPng} /> 
+
+
+<br />
+
+Execution-layer clients run on execution-layer (EL) networks, while consensus-layer clients run on consensus-layer (CL) networks. Every execution-layer network has a corresponding consensus-layer network. In this quickstart, you'll learn how to run a node + validator on the following network pairs:
+
+<table>
+    <tr>
+        <th style={{minWidth: 180 + 'px'}}>EL network</th> 
+        <th style={{minWidth: 180 + 'px'}}>CL network</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+      <td>Mainnet</td>
+      <td>Mainnet</td>
+      <td>TODO</td>
+    </tr> 
+    <tr>
+      <td>Goerli</td>
+      <td>Prater</td>
+      <td>TODO</td>
+    </tr>
+    <tr>
+      <td>Ropsten</td>
+      <td>Ropsten</td>
+      <td>Execution-layer Ropsten is deprecated, but is being used to test The Merge. Consensus-layer Ropsten is a new network that's helping us test The Merge with execution-layer Ropsten...</td>
+    </tr>
+</table>
+
+
+
+## Step 2: Identify your goals and system requirements
```

Comment:
I reverted this to "Identify your..." based on the H2 pattern of `## Step n: Action/Verb`. Would love to keep this consistent between sections.